src/mlpack/methods/isomap/isomap_main.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * @file isomap_main.cpp
+ * @author Rishabh Ranjan
+ * 
+ * Main executable to run Isomap
+ * 
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+
+#include "isomap.hpp"
+
+using namespace mlpack;
+using namespace mlpack::isomap;
+using namespace mlpack::util;
+using namespace std;
+
+PROGRAM_INFO("Isomap",
+    // Short Description
+    "An implementation of Isomap algorithm for dimensionality reduction. "
+    "Given an input dataset the algorithm fits the dataset into a lower "
+    "dimensional space, assuming the data is embedded into a "
+    "higher dimensional space but belongs to a lower dimensional space.",
```

Comment:
I think this is a good idea. I will try and make it better. It can also include classical multidimensional scaling like, 

"... using the k-nearest-neighbor graph for finding geodesic distances and classical multidimensional scaling for spectral decomposition."

I will try and make comments better.